dockerfile-generator.rb

```diff
@@ -0,0 +1,392 @@
+#!/usr/bin/env ruby
```

Comment:
They don't need any dependencies it's on the maintainer of this repo to add things to `Dockerfile.yml` and then run the script to generate the `Dockerfile` and people who want to use it they already have the `Dockerfile` pretty and ready to use.